recipes/mkdocs-ivory/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
The sdist on pypi already has a license file in it with the name 'LICENSE'. This file is redundant.

Reply:
ok, I removed the LICENSE file. Thank you.